c++/src/capnp/cc_capnp_library.bzl

```diff
@@ -121,7 +121,7 @@ def cc_capnp_library(
         name = name,
         srcs = srcs_cpp,
         hdrs = hdrs,
-        deps = deps + ["@capnp-cpp//src/capnp:capnp_runtime"],
+        deps = deps + ["@capnp-cpp//src/capnp:capnp", "@capnp-cpp//src/capnp:capnp_runtime"],
```

Comment:
What's the story behind this line?

Reply:
Ugh, apologies, this got pulled in from the work I was building on top of this. The short answer is that we might be technically violating bazel's indirect inclusion rules: https://bazel.build/reference/be/c-cpp#hdrs. I'm uncertain and still figuring it out. In any case, shouldn't be in this PR.